content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -75,11 +75,122 @@ To setup unified service tagging in a containerized environment:
 
 #### Configuration
 
+##### Automatic version tagging for containerized environments
+You can use the `version` tag to [monitor deployments][17] and to identify faulty code deployments through [Automatic Faulty Deployment Detection][18].
+
+Datadog sets the `version` tag for you in the following priority order. If you manually set `version`, Datadog does not override your `version` value.
+
+| Priority         | Version Value |
+|--------------|------------|
+| 1    |  {your version value}       |
+| 2   | {image_tag}_{git_commit_sha}       |
```

Comment:
Do we want to clarify that it's only the first 7 digits of the SHA?

Reply:
Updated